### ISSUE TITLE
Getting errors at compile time Member access into incomplete type 'reactjuce::EcmascriptEngine::Pimpl'

### ISSUE THREAD

**Issue by duro** [nick-thompson/react-juce#283]
I posted this in the Discord, so forgive me if this is redundant.

I am trying to get a simple HelloWorld app going. I have the JS bundle built, but when I compile the plugin in XCode I get the following error all over the place:

`Member access into incomplete type 'reactjuce::EcmascriptEngine::Pimpl'`

Here is a Gist of the only two files in the JUCE C++ app:
https://gist.github.com/duro/b5cca6caf1aa4fb9b735bf59596d7c85#file-pluginprocessor-cpp-L166

**Comment by milan-markovic** [comment]
I've fixed it by adding REACTJUCE_USE_DUKTAPE=1 to "Preprocessor definitions" in Projucer. No docs on that one. Also, REACTJUCE_USE_HERMES=1 doesn't seem to work.

**Comment by nick-thompson** [comment]
> I've fixed it by adding REACTJUCE_USE_DUKTAPE=1 to "Preprocessor definitions" in Projucer. No docs on that one. Also, REACTJUCE_USE_HERMES=1 doesn't seem to work.

Yep, that's exactly the fix (for now). This is an issue we intend to fix asap!

**Comment by milan-markovic** [comment]
@nick-thompson can you check why hermes isn't working?

**Comment by nick-thompson** [comment]
Of course, yea. I'm also going to ping @JoshMarler on this one as he's more knowledgeable on the details of the hermes integration
